sechub-pds-solutions/owasp-zap/docker/run.sh

```diff
@@ -50,7 +50,7 @@ localserver () {
         echo " * Accesskey: $S3_ACCESSKEY"
     fi
 
-    nohup owasp-zap -daemon -host 127.0.0.1 -port 8080 -config api.key="$ZAP_API_KEY" >> /dev/null &
+    nohup owasp-zap -daemon -silent -nostdout -host 127.0.0.1 -port 8080 -config api.key="$ZAP_API_KEY" &
```

Comment:
Please add a comment regarding the `-silent` option. The `-silent` option disables the telemetry collection of the call home addon: https://www.zaproxy.org/docs/desktop/addons/call-home/

That is an important topic.